app/models/proposal.rb

```diff
@@ -33,6 +33,7 @@ class Proposal < ActiveRecord::Base
   has_many :steps
   has_many :individual_steps, ->{ individual }, class_name: 'Steps::Individual'
   has_many :approvers, through: :individual_steps, source: :user
+  has_many :completers, through: :individual_steps, source: :completer
```

Comment:
I would probably go with `step_user` and `step_completer` -- but I am fine with keeping as-is for now and reivisiting later